src/bin/vip.js

```diff
@@ -36,7 +36,7 @@ const runCmd = async function () {
 		.command( 'logs', 'Get logs from your VIP applications' )
 		.command( 'search-replace', 'Perform search and replace tasks on files' )
 		.command( 'slowlogs', 'Get slowlogs from your VIP applications' )
-		.command( 'db', 'Run operations on your VIP application database' )
+		.command( 'db', 'Access an environment database.' )
```

Comment:
Maybe? An "environment database" doesn't sound right to me.
```suggestion
		.command( 'db', "Access an environment's database." )
```